.github/actions/perform-deployment/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "perform-deployment",
+  "version": "1.0.0",
+  "description": "This action uses python module `agent_build/tools/environment_deployments.py` to perform some deployment which is defined in it. The main purpose of the action is to cache results of this deployment to the GitHub Actions cache. Since the deployments consist of  steps, and different deployments can use the same step, then it's more reasonable to cache each step separately. That means that the regular [@actions/cache](https://github.com/actions/cache)  is not enough, and we need to use more flexible JS scripting to call GitHub Actions caching dynamically.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
All the code is written by us, right? If so, we should use Apache 2.0 license.

Reply:
All, except dependencies :)